includes/classes/Authentication.php

```diff
@@ -47,6 +47,11 @@ public function __construct( $args ) {
 	 * @return array
 	 */
 	public function format_get_args( $args = array(), $context = array() ) {
+		if ( ! isset( $args['headers'] ) ) {
+			$args['headers'] = array();
+		}
+		$args['headers']['X-Distributor-Version'] = DT_VERSION;
```

Comment:
@peterwilsoncc I see we are repeating this multiple times in the code base. I am okay with explicitly adding this in the header, but I want to check if we can add this header automatically with a filter hook to any distributor-related call.